jbpm-bom/pom.xml

```diff
@@ -815,6 +815,12 @@
         <version>${version.org.kie}</version>
         <classifier>sources</classifier>
       </dependency>
+      <dependency>
+        <groupId>org.jbpm</groupId>
+        <artifactId>jbpm-wb-case-mgmt-showcase</artifactId>
+        <version>${version.org.kie}</version>
+        <type>war</type>
+      </dependency>
```

Comment:
@psiroky The showcase app is a demo that shows all the UI building blocks related to case management. The idea is that the app itself isn't supported but the individual blocks are.  The idea is that these can be consumed by custom apps.
As for publishing maybe @krisv can comment on that, I believe it could be just a link to nexus.
Regarding tests QA is already looking into that.